agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -46,8 +49,16 @@ const (
 	// Environment variables to setup resource location
 	envProgramData              = "ProgramData"
 	dockerCredentialSpecDataDir = "docker/credentialspecs"
+	servicesRegistryKeyRoot     = `System\CurrentControlSet\Services`
+	taskCredentialsRegistryKey  = `AmazonECSCCGPlugin`
+	RegKeyPathFormat            = `HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\AmazonECSCCGPlugin\%s`
```

Comment:
nit: can we change `RegKeyPathFormat`  to camelCase if it is not being used outside of this package?

Reply:
yes will do when I publish final version with units